scanner/app.py

```diff
@@ -0,0 +1,80 @@
+from flask import Flask, jsonify, request
+from apscheduler.schedulers.background import BackgroundScheduler
+from datetime import datetime
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import text
+import subprocess
+import mysql.connector
+import requests
+import os
+import time
+from dotenv import load_dotenv
+import threading
+from nftScanner import main
+import logging
+
+# load environment variables from .env file
+load_dotenv()
+
+db_host = os.environ.get('DB_HOST')
+db_user = os.environ.get('DB_USER')
+db_password = os.environ.get('DB_PASSWORD')
+db_name = os.environ.get('DB_NAME')
+
+app = Flask(__name__)
+
+# setup SQL Alchemy
+app.config['SQLALCHEMY_DATABASE_URI'] = f'mysql://{db_user}:{db_password}@{db_host}/{db_name}'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+# Define the NFTOwnership model
+class NFTOwnership(db.Model):
```

Comment:
Do not create any orm within the app.py; the app.py should only initialize the server.